src/test/java/coydir/storage/JsonAdaptedPersonTest.java

```diff
@@ -93,15 +100,6 @@ public void toModelType_invalidPhone_throwsIllegalValueException() {
         assertThrows(IllegalValueException.class, expectedMessage, person::toModelType);
     }
 
-    @Test
-    public void toModelType_nullPhone_throwsIllegalValueException() {
```

Comment:
Wait why remove these test cases?

Reply:
I am removing the null cases so I just had a overall test case encompass all the N/A cases

Reply:
I think there's a bit of a mix-up here. `null` does not refer to "N/A". Those are two different values, and thus two different cases that deserve their own tests. Would it not be better to keep both?